src/app/views/communities/view/overview/edit/moderators/moderators.ts

```diff
@@ -0,0 +1,97 @@
+import Component from 'vue-class-component';
+import { arrayRemove } from '../../../../../../../utils/array';
+import { Api } from '../../../../../../../_common/api/api.service';
+import AppCardListAdd from '../../../../../../../_common/card/list/add/add.vue';
+import AppCardListItem from '../../../../../../../_common/card/list/item/item.vue';
+import AppCardList from '../../../../../../../_common/card/list/list.vue';
+import { Collaborator } from '../../../../../../../_common/collaborator/collaborator.model';
+import { Growls } from '../../../../../../../_common/growls/growls.service';
+import { ModalConfirm } from '../../../../../../../_common/modal/confirm/confirm-service';
+import {
+	BaseRouteComponent,
+	RouteResolver,
+} from '../../../../../../../_common/route/route-component';
+import FormCommunityCollaborator from '../../../../../../components/forms/community/collaborator/collaborator.vue';
+import { RouteStore, RouteStoreModule } from '../edit.store';
+
+@Component({
+	name: 'RouteCommunitiesViewEditModerators',
+	components: {
+		AppCardList,
+		AppCardListItem,
+		AppCardListAdd,
+		FormCommunityCollaborator,
+	},
+})
+@RouteResolver({
+	deps: { params: ['id'] },
+	resolver({ route }) {
+		return Api.sendRequest('/web/dash/communities/collaborators/' + route.params.id, {});
+	},
+})
+export default class RouteCommunitiesViewEditModerators extends BaseRouteComponent {
+	@RouteStoreModule.State
+	community!: RouteStore['community'];
+
+	@RouteStoreModule.State
+	collaboration!: RouteStore['collaboration'];
+
+	collaborators: Collaborator[] = [];
+	activeCollaborator: Collaborator | null = null;
+	isShowingCollaboratorAdd = false;
+
+	readonly Collaborator = Collaborator;
+
+	get isOwner() {
+		// The owner's collaboration is not returned from backend.
+		return this.collaboration === null;
+	}
```

Comment:
It would be cool to put this in the edit store instead of repeating it everywhere. Mostly because the decision to distinguish between owner and collaborators is lack of collaboration entry while owners DO have a collaboration entry, just with the owner role is too arbitrary...